usr/src/uts/aarch64/sys/archsystm.h

```diff
@@ -66,6 +72,61 @@ extern void reset(void) __NORETURN;
 
 #define	BLOCKZEROALIGN (4 * sizeof (void *))
 
+/*
+ * Mask interrupts, returning the old interrupt mask.
+ */
+static inline uint64_t
```

Comment:
We usually use `extern __GNU_INLINE` for things like this, so they're never not inline.   The original arm code didn't though.  I'm trying to fix them slowly, and don't mind if you don't change it.

Reply:
Pushed a new version with this addressed.